lib/cn/cn_tree.c

```diff
@@ -661,6 +661,21 @@ cn_tree_view_create(struct cn *cn, struct table **view_out)
             s->nodeid = kvset_get_nodeid(kvset);
             s->eklen = 0;
 
+            w = kvset_get_work(kvset);
+            if (w) {
+                ulong tm = (now - w->cw_t0_enqueue) / NSEC_PER_SEC;
+
+                n = snprintf(s->ekbuf, sizeof(s->ekbuf),
+                             "- %s/%s %lu:%02lu %s",
+                             cn_action2str(w->cw_action),
+                             cn_rule2str(w->cw_rule),
+                             (tm / 60) % 60, tm % 60,
+                             sts_job_wmesg_get(&w->cw_job));
+            }
+
+            if (!w || n < 1)
+                strlcpy(s->ekbuf, "-", sizeof(s->ekbuf));
+
```

Comment:
I have needed and wanted this forever!  So cool to obtain the tree view and see all the active transforms in progress!